### PR TITLE
Use recon API to get dcid from place id

### PR DESCRIPTION
**Pull request by shifucun**
This completely removes the need of using placeid2dcid.json in website! A very big simplification for maintenance

### CODE REVIEW

File: server/configmodule.py [hunk 1]
```diff
@@ -14,7 +14,8 @@ class Config:
     VERSION = '{}-{}'.format(os.environ.get('WEBSITE_HASH'),
                              os.environ.get('MIXER_HASH'))
     API_ROOT = 'http://127.0.0.1:8081'  # Port for Kubernetes ESP.
-    GCS_BUCKET = os.environ.get('GCS_BUCKET') or ''
+    # TODO: Use prod recon api in prod.
+    RECON_API_URL = 'https://autopush.recon.datacommons.org/id/resolve'
```
Comment (shifucun) [inline server/configmodule.py:18]: @Spaceenter can we use prod recon service for prod website now?

**Comment by shifucun** [comment]
@Spaceenter @pradh can you take a look at this change too?

File: server/configmodule.py (hunk 1 above)
Comment (pradh) [reply server/configmodule.py:18]: +1 to using prod here.

The recon service also uses the BT cache, and stores a version.

Suppose we make a DCID map change, what's the rollout process?  For example, during go/dcrelease, should we do something like this:

1. Send PR to update the cache version in recon service
2. Push to staging recon service
3. Validate staging website (which uses staging.recon)
4. Push to prod recon

**Review by pradh — COMMENTED** [review]
Nice change!

File: server/configmodule.py (hunk 1 above)
Comment (shifucun) [reply server/configmodule.py]: Updated mixer to serve recon APIs here as well. So there is no special handling for the recon APIs.

**Comment by shifucun** [comment]
Finally updated this PR now as recon service is merged into mixer.

**Review by beets — APPROVED** [review]
thanks bo!

File: server/configmodule.py [hunk 2]
```diff
@@ -14,7 +14,7 @@ class Config:
     VERSION = '{}-{}'.format(os.environ.get('WEBSITE_HASH'),
                              os.environ.get('MIXER_HASH'))
     API_ROOT = 'http://127.0.0.1:8081'  # Port for Kubernetes ESP.
-    GCS_BUCKET = os.environ.get('GCS_BUCKET') or ''
+    RECON_API_ROOT = 'http://127.0.0.1:8081'
```
Comment (pradh) [inline server/configmodule.py:17]: I see, so this is the instance of Recon+Mixer API service running co-located with Website?

**Review by pradh — APPROVED** [review]
Very nice!

(We should also update release doc)

File: server/configmodule.py (hunk 2 above)
Comment (shifucun) [reply server/configmodule.py:17]: Yes, that's right.

**Pull request merged**